### PR TITLE
Create Basic View for CreateEventQuery

### PR DESCRIPTION
**Pull request by tommartensen**

### CODE REVIEW

File: src/CreateEventQueryView/CreateEventQueryView.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import Header from './../Header';
+import { css } from 'aphrodite';
+import EventTypeInformation from "./EventTypeInformation";
+import EventQueryInputArea from "./EventQueryInputArea";
+import config from "../config/config";
+import AppStyles from "../AppStyles";
+
+class CreateEventQueryView extends ConnectionComponent {
+
+    static getButtons() {
```
Comment (friedow) [inline src/CreateEventQueryView/CreateEventQueryView.js]: Why are the buttons extracted to a method? This adds no extra logic.

File: src/CreateEventQueryView/EventQueryInputArea.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React, {Component} from "react";
+import {TextField} from "material-ui";
+import config from "../config/config";
+
+class EventQueryInputArea extends Component {
```
Comment (friedow) [inline src/CreateEventQueryView/EventQueryInputArea.js:5]: Why is this a component? This adds no extra logic.

File: src/CreateEventQueryView/CreateEventQueryView.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        return (
+            <div>
+                <Header title={"Create Event Query for " + eventType.Name}/>
+                <Container>
+                    <Row>
+                        <Col md={4}>
+                            <EventTypeInformation attributes={attributes}/>
+                        </Col>
+                        <Col md={8}>
+                            <EventQueryInputArea/>
```
Comment (frederike-ramin) [inline src/CreateEventQueryView/CreateEventQueryView.js]: The area is not really an area but rather a line, that's a bit irritating to be honest.
Also there is no Desription text what is supposed to be put in there so my gueass was that one should hand the timestamp parameter as it is aligned with that atribute.
Please overthik that styling.

File: src/CreateEventQueryView/EventTypeInformation.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React, {Component} from "react";
+import {List, ListItem} from "material-ui/List";
+import {css} from "aphrodite";
+
+class EventTypeInformation extends Component {
+    static generateListItem(attribute, key) {
```
Comment (friedow) [inline src/CreateEventQueryView/EventTypeInformation.js]: Why is this extracted to a method? This adds no extra functionality.

**Review by frederike-ramin — CHANGES_REQUESTED** [review]
Haven't looked at all of it, but this should be enough work for now. ;)

File: src/Header.js [hunk 5]
```diff
@@ -35,7 +35,7 @@ class Header extends Component {
         if (pageLocation === "settings") {
             iconElementRight = <IconButton href="/create/eventType"><IconAdd/></IconButton>;
         }
-        if (pageLocation === "createEventType") {
+        if (pageLocation === "create") {
```
Comment (friedow) [inline src/Header.js]: This lacks readability. Please insert a comment here.

File: src/Utils/MockData/BackendMock.js [hunk 6]
```diff
@@ -87,6 +88,19 @@ class BackendMock {
         return associatedEventTypes;
     }
 
+    static getEventType(params) {
```
Comment (friedow) [inline src/Utils/MockData/BackendMock.js]: Please reformat this using tabs as indentation.

File: src/config/config-template.js [hunk 7]
```diff
@@ -16,7 +16,8 @@ const config = {
 		error: "#B71C1C",
 	},
 	descriptions: {
-		textNumberOfEvents: "Number of events"
+		textNumberOfEvents: "Number of events",
+        exampleQuery: "INSERT INTO ExampleType SELECT * FROM PATTERN [[2] SimpleType]"
```
Comment (friedow) [inline src/config/config-template.js]: missing indentation

**Review by friedow — CHANGES_REQUESTED** [review]

File: src/CreateEventQueryView/CreateEventQueryView.js (hunk 1 above)
Comment (tommartensen) [reply src/CreateEventQueryView/CreateEventQueryView.js]: ok

File: src/CreateEventQueryView/EventQueryInputArea.js (hunk 2 above)
Comment (tommartensen) [reply src/CreateEventQueryView/EventQueryInputArea.js:5]: The new component will be extended into the Simple Esper UI.

File: src/Header.js (hunk 5 above)
Comment (tommartensen) [reply src/Header.js]: ok

File: src/Utils/MockData/BackendMock.js (hunk 6 above)
Comment (tommartensen) [reply src/Utils/MockData/BackendMock.js]: ok

File: src/CreateEventQueryView/CreateEventQueryView.js (hunk 3 above)
Comment (tommartensen) [reply src/CreateEventQueryView/CreateEventQueryView.js]: restyled page to make this more clear, thank you for the feedback

**Review by frederike-ramin — APPROVED** [review]
Nice!

File: src/CreateEventQueryView/EventTypeInformation.js (hunk 4 above)
Comment (tommartensen) [reply src/CreateEventQueryView/EventTypeInformation.js]: i oriented myself on your component in detail view...

File: src/config/config-template.js (hunk 7 above)
Comment (tommartensen) [reply src/config/config-template.js]: that spaces/tab thing again...

**Review by friedow — APPROVED** [review]

**Pull request merged**